configs/_base_/models/tinyvit/tinyvit-21m-384.py

```diff
@@ -0,0 +1,25 @@
+# Model settings
+model = dict(
+    type='ImageClassifier',
+    backbone=dict(
+        type='TinyViT',
+        arch='21m',
+        img_size=(384, 384),
+        window_size=[12, 12, 24, 12],
```

Comment:
I think we can only reserve 224 version models and modify the `window_sizes` in the final config file.

Reply:
And the `224` or `384` can be removed from the filenames.